baremetal/metal3machine_manager.go

```diff
@@ -1143,6 +1146,14 @@ func (m *MachineManager) SetError(message string, reason capierrors.MachineStatu
 	m.Metal3Machine.Status.FailureReason = &reason
 }
 
+func (m *MachineManager) SetConditionMetal3MachineToFalse(t capi.ConditionType, reason string, severity capi.ConditionSeverity, messageFormat string, messageArgs ...interface{}) {
+	conditions.MarkFalse(m.Metal3Machine, t, reason, severity, messageFormat, messageArgs...)
+}
+
+func (m *MachineManager) SetConditionMetal3MachineToTrue(t capi.ConditionType) {
+	conditions.MarkTrue(m.Metal3Machine, t)
+}
```

Comment:
I might be missing something, why not do directly refer `conditions.MarkTrue/conditions.MarkFalse` ?

Reply:
@fmuyassarov, because `reconcileNormal` takes the interface as input. 
```
reconcileNormal(ctx context.Context,
	machineMgr baremetal.MachineManagerInterface,
```